CHANGES.md

```diff
@@ -15,7 +15,13 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Move canarie-api configuration for cowbird from proxy to canarie-api config directory
+  - The canarie-api configuration for cowbird was being loaded whenever the proxy component was enabled instead
+    of when the canarie-api component was enabled. Since these components can now be enabled separately, the
+    configuration has to be moved to ensure that canarie-api configuration files aren't unintentionally mounted
+    to a container that is just running an nginx proxy.
```

Comment:
Add a reference to https://github.com/Ouranosinc/CanarieAPI/pull/18 + mention of requirement `CanarieAPI>=1.0.0` to accomplish this.

Reply:
Ok if you want. But this PR has nothing to do with that change. It moves a file that I should have moved in #296 and has been raising warning messages ever since nginx can be run without canarie-api in #310. 

My vote would be to merge this now since it is entirely independent of https://github.com/Ouranosinc/CanarieAPI/pull/18

Reply:
Oh. By "Since these components can now be enabled separately", it thought you were refering to the nginx proxy and canarie-api being enabled separately. If you were refering to the service components of birdhouse-deploy, then you can ignore this comment.